storage/rocksdb/rdb_i_s.cc

```diff
@@ -845,17 +845,22 @@ static int rdb_i_s_compact_stats_fill_table(
       continue;
     }
 
-    std::map<std::string, double> props;
+    std::map<std::string, std::string> props;
     bool bool_ret MY_ATTRIBUTE((__unused__));
     bool_ret = rdb->GetMapProperty(cfh, "rocksdb.cfstats", &props);
     DBUG_ASSERT(bool_ret);
 
     for (auto const &prop_ent : props) {
+      std::string prefix = "compaction.";
```

Comment:
Since this doesn't change, can probably declare this outside the for loop and make it const.